pyomo/gdp/basic_step.py

```diff
@@ -19,11 +19,12 @@
 logger = logging.getLogger('pyomo.gdp')
 
 
-def _pseudo_clone(self):
-    """Clone everything in a Disjunct except for the indicator_var"""
+def _clone_all_but_indicator_vars(self):
+    """Clone everything in a Disjunct except for the indicator_vars"""
```

Comment:
Can a Disjunct have more than one indicator_var ?

Reply:
No, but with this PR, it has `binary_indicator_var` and `indicator_var`, which is why I made the description plural.